app/controllers/v1/stored_mails_controller.rb

```diff
@@ -1,14 +1,21 @@
 class V1::StoredMailsController < V1::ApplicationController
   before_action :set_model, only: %i[accept reject]
 
-  def accept
+  def accept # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
     authorize @model
 
     if @model.mailgun_mail?
       MailForwardJob.perform_later(@model.mail_alias, @model.message_url)
     else
       # :nocov:
-      MailImprovmxForwardJob.perform_later(@model)
+      to_addresses = @model.mail_alias.mail_addresses
```

Comment:
It might be better to put this loop in the job, and the create a separate job which does the individual sending. If there are 100 mail addresses it might take quite some time to schedule everything. And it is also easier to create a spec for it when it is in a job

Reply:
I have added an extra job. I am also using the default SMTP sender now instead. Could you verify this will work as intended?

Reply:
The code seems fine, did you test it? Then I think we can deploy it. I do not really have time to test it in the coming week